migrate/loader.go

```diff
@@ -0,0 +1,162 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var loaderCommand = &cobra.Command{
+	Use:   "loader [version|latest|recommended]",
+	Short: "Migrate your modloader version to a newer version.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		modpack, err := core.LoadPack()
+		if err != nil {
+			// Check if it's a no such file or directory error
+			if os.IsNotExist(err) {
+				fmt.Println("No pack.toml file found, run 'packwiz init' to create one!")
+				os.Exit(1)
+			}
+			fmt.Printf("Error loading pack: %s\n", err)
+			os.Exit(1)
+		}
+		// Get our current loader, would use the modpack.GetLoaders function
+		// but, it supplements Quilt with Fabric, which isn't needed for this
+		var currentLoaders []string
+		// Add all the keys from modpack.Versions
+		for key := range modpack.Versions {
+			// Ignore the "minecraft" loader
+			if key == "minecraft" {
+				continue
+			}
+			currentLoaders = append(currentLoaders, key)
+		}
+		// Do some sanity checks on the current loader slice
+		if len(currentLoaders) == 0 {
+			fmt.Println("No loader is currently set in your pack.toml!")
+			os.Exit(1)
+		} else if (!slices.Contains(currentLoaders, "quilt") && len(currentLoaders) > 1) || (slices.Contains(currentLoaders, "quilt") && len(currentLoaders) > 2) {
```

Comment:
I don't think this needs to special-case quilt anymore :)

Reply:
Ah yeah you right lol